src/data/set/pointwise/smul.lean

```diff
@@ -449,6 +472,19 @@ pairwise_disjoint_image_right_iff $ λ _ _, mul_right_injective _
 
 end left_cancel_semigroup
 
+section monoid
+variables [monoid α] [mul_action α β]
+
+@[to_additive] lemma op_smul_set_smul_eq_smul_smul_set (s : set α) (a : α) (t : set β) :
+  (op a • s) • t = s • a • t :=
+by simpa using mul_smul s {a} t
+
+@[to_additive] lemma op_smul_set_mul_eq_mul_smul_set (s : set α) (a : α) (t : set α) :
+  (op a • s) * t = s * a • t :=
```

Comment:
Incidentally this statement is the missing typeclass for some bimodule problems (generalizing `set α` to some arbitrary `β`).

Reply:
Yeah, I mentioned that to you on Zulip already. I decided to roll along with the special case because
1. The proof is trivial
2. The typeclass problem is very much not